### PR TITLE
Css reset and increase css specificity for order_stash plugin

### PR DESCRIPTION
**Pull request by kosdiamantis**
Spot the difference!
v1vsv2

### CODE REVIEW

File: src/plugins/order_stash.coffee [hunk 1]
```diff
@@ -176,31 +181,36 @@ class OrderStash
     font-style: italic;
     line-height: 13px;
     box-sizing: border-box;
+
+    transition: none;
```
Comment (Zorbash) [inline src/plugins/order_stash.coffee]: Why `transition: none` here?

File: src/plugins/order_stash.coffee [hunk 2]
```diff
@@ -176,31 +181,36 @@ class OrderStash
...
+    display: inline-block;
     margin-bottom: 23px;
     line-height: 17px;
     font-family: Verdana, Arial, sans-serif;
     font-size: 13px;
   }
 
-  #sa-order-stash-rationale p.sa-order-stash-privacy {
+  #sa-order-stash-plugin #sa-order-stash-rationale p.sa-order-stash-privacy {
+    font-style: italic;
```
Comment (Zorbash) [inline src/plugins/order_stash.coffee]: ?

File: src/plugins/order_stash.coffee (hunk 1 above)
Comment (kosdiamantis) [inline src/plugins/order_stash.coffee]: I did not add this one :smile: it was already defined here: https://github.com/skroutz/analytics.js/pull/20/files#diff-7316f697d60038fadbc42b52a47928d0L164

File: src/plugins/order_stash.coffee (hunk 2 above)
Comment (kosdiamantis) [inline src/plugins/order_stash.coffee]: Same here it was already defined here: https://github.com/skroutz/analytics.js/pull/20/files#diff-7316f697d60038fadbc42b52a47928d0L104

**Comment by Zorbash** [comment]
:+1: LGTM :shipit:

**Pull request merged**